src/helics/core/BasicHandleInfo.hpp

```diff
@@ -58,6 +58,7 @@ class BasicHandleInfo
     const std::string key;  //!< the name of the handle
     const std::string type;  //!< the type of data used by the handle
     const std::string units;  //!< the units associated with the handle
+    const std::string interface_info;
```

Comment:
this should not be const,  the others are because they cannot change once constructed,  info should be able to like `used` and `flags`